updater/Gemfile.lock

```diff
@@ -381,4 +388,4 @@ DEPENDENCIES
   webmock (~> 3.18)
 
 BUNDLED WITH
-   2.4.22
+   2.3.26
```

Comment:
You probably used Bundler 2.3.26 to regenerate the lockfile. Make sure you use Bundler 2.4.22 so this does not get downgraded. If you run `bundler` within the development image that should be the case already but if you run `BUNDLE_GEMFILE=updater/Gemfile bundle lock` outside of the development container, you'll need `gem install bundler:2.4.22` before that.

Reply:
updated 👍